content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -402,6 +403,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
    Check [Service Account Token Volumes](https://git.k8s.io/community/contributors/design-proposals/storage/svcacct-token-volume-source.md)
    for more details.
 - `ConfigurableFSGroupPolicy`: Allows user to configure volume permission change policy for fsGroups when mounting a volume in a Pod. See [Configure volume permission and ownership change policy for Pods](/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods) for more details.
+- `CronJobControllerV2`: Enable alternative implementation of the CronJob controller.
```

Comment:
nit (doesn't block merge):
```suggestion
- `CronJobControllerV2`: Use an alternative implementation of the
   {{< glossary_tooltip text="CronJob" term_id="cronjob" >}} controller. Otherwise,
   version 1 of the same controller is selected. The version 2 controller provides
   experimental performance improvements.
```
?

Reply:
:+1: